From 8d530d9ef94ab94d7e23301dc79f9ea30d8ad392 Mon Sep 17 00:00:00 2001
From: Jeffrey Ladouceur <Jeffrey.Ladouceur@freescale.com>
Date: Thu, 17 Apr 2014 18:07:47 -0400
Subject: [PATCH 0976/1383] fsl_pme: remove incorrect usage of flag to
 request_irq

The hardware doesn't support affinity on individual error interrupts.
Therefore it's not possible to remove correctly remove the warning
"Cannot set affinity for irq" since this is a reflection of reality.

Signed-off-by: Jeffrey Ladouceur <Jeffrey.Ladouceur@freescale.com>
Change-Id: Ide2a507c60dc1e6b4be1c5f0aba352a37071ae80
Reviewed-on: http://git.am.freescale.net:8181/#/c/11126/
Reviewed-on: http://git.am.freescale.net:8181/11213
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Scott Wood <scottwood@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl_pme2/pme2_ctrl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_pme2/pme2_ctrl.c b/drivers/staging/fsl_pme2/pme2_ctrl.c
index dba6cc9..0df0e8c 100644
--- a/drivers/staging/fsl_pme2/pme2_ctrl.c
+++ b/drivers/staging/fsl_pme2/pme2_ctrl.c
@@ -339,7 +339,7 @@ static int of_fsl_pme_probe(struct platform_device *ofdev)
 	if (likely(pme_err_irq != NO_IRQ)) {
 		/* Register the pme ISR handler */
 		err = request_irq(pme_err_irq, pme_isr,
-			IRQF_SHARED | IRQF_PERCPU, "pme-err", dev);
+			IRQF_SHARED, "pme-err", dev);
 		if (err) {
 			dev_err(dev, "request_irq() failed\n");
 			goto out_unmap_ctrl_region;
-- 
2.8.1

